src/App.js

```diff
@@ -1,11 +1,30 @@
-import './App.css';
+import "./App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { Registration } from "./pages/Registration";
+import React from "react";
+import { HomePage } from "./pages/HomePage";
+import { Container } from "./components/Container";
+import { Navbar } from "./components/Navbar";
+import HotelPage from "./pages/HotelPage/HotelPage";
+import { Provider } from "react-redux";
+import { store } from "./store";
 
 const App = () => {
   return (
-    <div className="App">
-
-    </div>
+    // eslint-disable-next-line react/react-in-jsx-scope
+    <BrowserRouter>
+      <Container>
+        <Navbar />
+      </Container>
+      <Provider store={store}>
+        <Routes>
+          <Route path="/registration" element={<Registration />} />
+          <Route path="/" element={<HomePage />} />
+          <Route path="/hotels/:id" element={<HotelPage />} />
```

Comment:
It would be better to use PATH const instead of strings
```suggestion
          <Route path={PATH.login} element={<Registration />} />
          <Route path={PATH.index} element={<HomePage />} />
          <Route path={PATH.hotelById} element={<HotelPage />} />
```